audformat/core/utils.py

```diff
@@ -12,11 +12,12 @@
 import audeer
 import audiofile
 
+from audformat.core import common
```

Comment:
Would also be fine with me if you do
```python
from audformat.core.common import to_audformat_dtype
```
as we usually do.

The main reason I did not do it inside `audformat.core.scheme` was that there we define a method that is also called `to_pandas_dtype()`, and I thought `common.to_pandas_dtype()` will then be more obvious to differentiate them.

Reply:
Ok, changed it accordingly.